### PR TITLE
Update blog post themes

### PR DESCRIPTION
**Pull request by steffilazerte**
Fixes (#189)

### CODE REVIEW

File: posts/posts.csv [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+Using an rOpenSci resource, such as the Dev Guide or a task view;Getting your toes wet in R: Hydrology, meteorology, and more;https://ropensci.org/blog/2019/04/02/hydrology-task-view/;
+About a peer-reviewed package developed by the authors;rmangal: Making Ecological Networks Easily Accessible;https://ropensci.org/blog/2019/10/21/rmangal/;talks through the scientific problem and context, shows some code examples, and talks about peer review but doesn't make that the dominant part of the post.
+About a peer-reviewed package developed by the authors;Forcing Yourself to Make Your Life Easier;https://ropensci.org/blog/2018/04/12/ijtiff/;is an honest post with some reflection and an important message.
+About a peer-reviewed package developed by the authors;The av Package: Production Quality Video in R;https://ropensci.org/technotes/2018/10/06/av-release/;(tech note) is to the point. av is not a peer-reviewed package but this is a good example of a tech note.
+Major updates to key packages;(Re)introducing skimr v2 - A year in the life of an open source R project;https://ropensci.org/blog/2019/10/29/skimrv2/);
```
Comment (maelle) [inline posts/posts.csv:17]: Should we rephrase "key" to something like "generalist"?

**Review by maelle — APPROVED** [review]
Thank you! I only have a suggestion/question.

File: posts/posts.csv (hunk 1 above)
Comment (steffilazerte) [reply posts/posts.csv:17]: Hmm, good point, what about "popular"? That's a bit weird though, "important" isn't much better. 

Perhaps, "Major updates to packages with broad audiences"? a bit less "are you in the popular crowd?"

Comment (maelle) [reply posts/posts.csv:17]: > Major updates to packages with broad audiences

I like that!

**Pull request merged**